test/lib/slim-rails_assets_test.rb

```diff
@@ -20,7 +20,7 @@ class DummyApp < ::Rails::Application
 
       asset_path = File.join(dir, 'app', 'assets', 'html', 'test.slim')
       FileUtils.mkdir_p(File.dirname(asset_path))
-      File.write(asset_path, ".test\n  | hi")
+      File.write(asset_path, %(div class=logical_path\n  | hi))
```

Comment:
@yasaichi why are we changing the test from one that parses slim `.test` code for div, to explicit div?

Reply:
@januszm 
I'm sorry for confusing you.
This change is for the following purposes:

* To check whether the asset helpers work or not
* To keep assertions simple

To accomplish the aboves, I replaced `.test` with `div class=logical_path` (as you may know, the method returns  `"test"`).

If you think it's difficult to understand, I'll make it easier by the other way, for example adding another assertion.

Reply:
It would be best if we keep the original test and add new for your test case. Then I guess we could merge it and prepare for relase 3.1.4 or 3.2.0 depending on how big the new feature is.

Reply:
@januszm 
OK, I'll do that within a few days.